src/components/views/settings/index.tsx

```diff
@@ -163,7 +178,21 @@ export const Settings: React.FC<SettingsProps> = ({navigation}) => {
                   accessibilityLabel={label}
                   accessibilityRole="button"
                   accessibilityHint={hint}
-                  onPress={() => navigation.navigate(screen)}>
+                  onPress={() =>
+                    screen === 'test'
+                      ? Linking.openURL(
+                          `mailto:support@example.com?subject=App Feedback for COVID Alert NY: Version ${version}&body=\n\n\n\n
+                          Device: ${getModel()}
+                          OS version: ${Platform.Version}
+                          Status of EN framework: ${exposure.enabled}
```

Comment:
@floridemai will know best but I think this should be one of the status strings not a bool, one or both of `exposure.status.state` or `exposure.permissions.exposure.status`?

Reply:
@AlanSl @JohnAGill I don't think we should follow exactly what the ticket says but send out info that is actually useful to us. What I'd do is:
Closeness sensing status: (!exposure.enabled || exposure.status.state !== StatusState.active || exposure.permissions.notifications.status !== PermissionStatus.Allowed) ? 'inactive' : 'active' (you can revert it if you want, but do include notifications)
ENS Details: JSON.stringy of all the ENS variable that we have. You can do {"c":canSupport, "s":supported, "st":status...)
Notifications: exposure.permissions.notifications.status !== PermissionStatus.Allowed ? Allowed : Not Allowed